### PR TITLE
Converting build from project.json to csproj

### PR DESCRIPTION
**Pull request by anmenaga**
Build was failing due to .NET tools moved from `project.json` to `csproj`.
Conversion was done using `dotnet migrate`.
Build completes successfully after the conversion.

### CODE REVIEW

File: src/PowerShell.Cmdletization.OData/PowerShell.Cmdletization.OData.csproj [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    <Description>Generates a module that contains cmdlets to manage an OData endpoint</Description>
+    <TargetFrameworks>netstandard1.6;net451</TargetFrameworks>
+    <AssemblyName>PowerShell.Cmdletization.OData</AssemblyName>
+    <OutputType>Library</OutputType>
+    <PackageId>PowerShell.Cmdletization.OData</PackageId>
+    <PackageTargetFallback Condition=" '$(TargetFramework)' == 'netstandard1.6' ">$(PackageTargetFallback);dnxcore50;portable-net45+win8</PackageTargetFallback>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.PowerShell.SDK" Version="1.0.0-alpha10" />
```
Comment (SteveL-MSFT) [inline src/PowerShell.Cmdletization.OData/PowerShell.Cmdletization.OData.csproj]: should we update this version to newer one?

File: src/ResGen/ResGen.csproj [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    <TargetFramework>netcoreapp1.0</TargetFramework>
+    <AssemblyName>resgen</AssemblyName>
+    <OutputType>Exe</OutputType>
+    <PackageId>ResGen</PackageId>
+    <RuntimeIdentifiers>ubuntu.16.04-x64;ubuntu.14.04-x64;debian.8-x64;centos.7-x64;win7-x64;win81-x64;win10-x64;osx.10.11-x64</RuntimeIdentifiers>
+    <RuntimeFrameworkVersion>1.0.4</RuntimeFrameworkVersion>
+  </PropertyGroup>
+
+  <ItemGroup Condition=" '$(TargetFramework)' == 'netcoreapp1.0' ">
+    <PackageReference Include="System.Xml.XDocument" Version="4.0.11" />
```
Comment (SteveL-MSFT) [inline src/ResGen/ResGen.csproj]: Should update to 4.3.0

File: src/PowerShell.Cmdletization.OData/PowerShell.Cmdletization.OData.csproj (hunk 1 above)
Comment (anmenaga) [reply src/PowerShell.Cmdletization.OData/PowerShell.Cmdletization.OData.csproj]: we should; but in a separate pr;
Opened issue #15  (Update references to the latest versions).

File: src/ResGen/ResGen.csproj (hunk 2 above)
Comment (anmenaga) [reply src/ResGen/ResGen.csproj]: Opened issue #15  (Update references to the latest versions).

**Pull request merged**